modules/reconciliation/view/list.php

```diff
@@ -0,0 +1,646 @@
+      }
+      print " " . $Reconciliation->voucherMessage($Voucher->VoucherID);
+?>
+          </td>
+          <td class="noprint">
+<?
+      $ButtonConfArray = array(
+        'url' => $_lib['sess']->dispatch . "t=reconciliation.list&AccountPlanID=" . $Reconciliation->AccountPlanID . "&amp;MatchAccountPlanID=" . $Voucher->AccountPlanID . "&amp;MatchKID=" . $Voucher->KID . "&amp;$showURL",
+        'name' => 'Vis samme kid'
+      );
+      print $_lib['form3']->button($ButtonConfArray);
+?>
+          </td>
+        </tr>
+<?
+    }
+?>
+        <tr>
+          <td colspan="18">
+          <td colspan="2">
+            <input type="checkbox" name="selectedClose[]" value="<? echo $AccountPlanID; ?>">
+<?
+    if ($Closeable) {
+      $ButtonConfArray = array(
+        'name' => 'action_reconciliation_closethis_' . $AccountPlanID,
+        'value' => 'Lukk denne',
+        'accesskey' => 'L'
+      );
+      print $_lib['form3']->submit($ButtonConfArray);
+    }
+    $ButtonConfArray = array(
+      'name' => 'action_reconciliation_openthis_' . $AccountPlanID,
+      'value' => '&Aring;pne denne'
+    );
+    print $_lib['form3']->submit($ButtonConfArray);
+    $ButtonConfArray = array(
+      'name' => 'action_reconciliation_update',
+      'value' => 'Lagre (S)',
+      'accesskey' => 'S'
+    );
+    print $_lib['form3']->submit($ButtonConfArray);
+?>
+          </td>
+        </tr>
+        <tr>
+          <th class="sub" colspan="4">Sum for konto <? print $AccountPlanID ?></th>
+          <th class="sub number align-right">
+<?
+    if ($Reconciliation->SumAccountH[$AccountPlanID]->Diff >= 0) {
+      print $_lib['format']->Amount($Reconciliation->SumAccountH[$AccountPlanID]->Diff);
+    }
+?>
+          </th>
+          <th class="sub number align-right">
+<?
+    if ($Reconciliation->SumAccountH[$AccountPlanID]->Diff < 0) {
+      print $_lib['format']->Amount($Reconciliation->SumAccountH[$AccountPlanID]->Diff);
+    }
+?>
+          </th>
+          <th class="sub number align-right">
+<?
+    if ($Reconciliation->SumAccountH[$AccountPlanID]->FDiff >= 0) {
+      print $_lib['format']->Amount($Reconciliation->SumAccountH[$AccountPlanID]->FDiff);
+    }
+?>
+          </th>
+          <th class="sub number align-right">
+<?
+    if ($Reconciliation->SumAccountH[$AccountPlanID]->FDiff < 0) {
+      print $_lib['format']->Amount($Reconciliation->SumAccountH[$AccountPlanID]->FDiff);
+    }
+?>
+          </th>
+          <th class="sub" colspan="12"></th>
+        </tr>
+<?
+  }
+?>
+
+<?
+  foreach ($Reconciliation->HiddingAccounts as $Account) {
+    $ShowAccountLink = $_SETUP['DISPATCH'] . "t=reconciliation.list&report_Sort=JournalID&AccountPlanID=$AccountPlanID&ReskontroFromAccount=$ReskontroFromAccount&ReskontroToAccount=$ReskontroToAccount&report.DepartmentID=$DepartmentID&report.ProjectID=$ProjectID&show_report_search=Kj%F8r+rapport&showOnly=" . $Account['AccountPlanID']; 
```

Comment:
Fixed, was using the same var name for `AccountPlanID` in the loop as it is the main param added to the link